File: CS/WpfApplication1/WizardCustomization/CustomSelectDataMemberPage.cs
using DevExpress.Data.XtraReports.ServiceModel;
using DevExpress.Data.XtraReports.Wizard;
using DevExpress.Data.XtraReports.Wizard.Presenters;
using DevExpress.Data.XtraReports.Wizard.Views;
using System;
// ...

namespace WpfApplication1.WizardCustomization {
    class CustomSelectDataMemberPage : SelectDataMemberPage<ReportModel> {
        public CustomSelectDataMemberPage(ISelectDataMemberPageView view, IReportWizardServiceClient client) 
            : base(view, client) { }

        public override Type GetNextPageType() {
            return typeof(CustomSelectColumnsPage);
        }
    }
}

File: VB/WpfApplication1/WizardCustomization/CustomSelectDataMemberPage.vb
Imports Microsoft.VisualBasic
Imports DevExpress.Data.XtraReports.ServiceModel
Imports DevExpress.Data.XtraReports.Wizard
Imports DevExpress.Data.XtraReports.Wizard.Presenters
Imports DevExpress.Data.XtraReports.Wizard.Views
Imports System
' ...

Namespace WpfApplication1.WizardCustomization
	Friend Class CustomSelectDataMemberPage
		Inherits SelectDataMemberPage(Of ReportModel)
		Public Sub New(ByVal view As ISelectDataMemberPageView, ByVal client As IReportWizardServiceClient)
			MyBase.New(view, client)
		End Sub

		Public Overrides Function GetNextPageType() As Type
			Return GetType(CustomSelectColumnsPage)
		End Function
	End Class
End Namespace
